client/components/common/DropDown.js

```diff
@@ -21,12 +21,11 @@ class DropDown extends Component {
       children,
       className,
     } = this.props;
-    const classNames = `dropdown ${className && className}`;
+    const classNames = `dropdown ${className}`;
```

Comment:
Nice insight here @damafeez. Did you consider the possibility that `className` may be undefined here and leads to a classname like `dropdown undefined`?

Reply:
Thank you for your feedback. This has been fixed.